check_tweets.py

```diff
@@ -0,0 +1,57 @@
+from mwrogue.esports_client import EsportsClient
+from mwrogue.esports_client import AuthCredentials
+import tweepy
+from datetime import date, timedelta
+import time
+from dotenv import load_dotenv
+import os
+import re
+
+load_dotenv()
+bearertoken = os.getenv("bearertoken")
+
+checkfrom = date.today() - timedelta(days=4)
+
+def main():
+	twitter_client = tweepy.Client(bearer_token=bearertoken)
+	credentials = AuthCredentials(user_file="me")
+	site = EsportsClient("lol", credentials=credentials)
+
+	response = site.cargo_client.query(
+		limit = "max",
+		tables = "NewsItems=NI",
+		fields = "NI.Source, NI._pageName=pageName, NI.N_LineInDate",
+		where = 'NI.Source IS NOT NULL AND NI.Date_Sort >= "{}"'.format(checkfrom),
+		order_by = "NI.Date_Sort DESC"
+	)
+
+	for source in response:
+		sourcelist = source["Source"].split(":::")
+		datapage = source["pageName"]
+		lineindate = source["N LineInDate"]
+		for source in sourcelist:
+			if not source:
+				continue
+			source = source.split(";;;")
+			if source[2] == "twitter.com":
+				splitStatus = re.search(r"status/([0-9]+)", source[0])[1]
```

Comment:
can you call `splitStatus` something like `tweet_id` (I think that's what it is right?) or if that's not what it is, something else that's more descriptive?